cypress/integration/article_spec.js

```diff
@@ -0,0 +1,21 @@
+describe('News Article', () => {
+const url = 'http://localhost:3000/';
+
+    it('should display 200/OK', () => {
+        cy.request(url + 'status').then((response) => {
+            expect(response.status).to.eq(200)
+        })
+    })
+
+    it('should render a headline', () => {
+        cy.visit(url)
+        cy.url().should('equal', url)
```

Comment:
Is this needed? I don't think there was an AC to test the URL format.

Reply:
Confirmed with James B